deploy/docs/Non_Helm_Installation.md

```diff
@@ -60,273 +39,153 @@ First, you'll need to set up the relevant [fields](https://help.sumologic.com/Ma
 - pod
 - service
 
-### Automatic Source Creation and Setup YAML
-
-This approach requires access to the Sumo Logic Collector API. It will create Kubernetes resources in your environment and run a container that uses the [Sumo Logic Terraform provider](https://github.com/SumoLogic/sumologic-terraform-provider) to create a Hosted Collector and multiple HTTP Sources in Sumo. It also uses the [Kubernetes Terraform provider](https://www.terraform.io/docs/providers/kubernetes/index.html) to create a Kubernetes secret to store the HTTP source endpoints to be used by Fluentd later.
-
-First, create the namespace. (We recommend using `sumologic` for easier setup.)
-```sh
-kubectl create namespace sumologic
-``` 
-
-Run the following command to download and apply the YAML file containing all the Kubernetes resources. Replace the `<NAMESPACE>`, `<SUMOLOGIC_ACCESSID>`, `<SUMOLOGIC_ACCESSKEY>`, `<COLLECTOR_NAME>` and `<CLUSTER_NAME>` variables with your values.
-
-```sh
-curl -s https://raw.githubusercontent.com/SumoLogic/sumologic-kubernetes-collection/release-v1.0/deploy/kubernetes/setup-sumologic.yaml.tmpl | \
-sed 's/\$NAMESPACE'"/<NAMESPACE>/g" | \
-sed 's/\$SUMOLOGIC_ACCESSID'"/<SUMOLOGIC_ACCESSID>/g" | \
-sed 's/\$SUMOLOGIC_ACCESSKEY'"/<SUMOLOGIC_ACCESSKEY>/g" | \
-sed 's/\$COLLECTOR_NAME'"/<COLLECTOR_NAME>/g" | \
-sed 's/\$CLUSTER_NAME'"/<CLUSTER_NAME>/g" | \
-tee setup-sumologic.yaml | \
-kubectl -n sumologic apply -f -
-```
-
-Run the following command to make sure the `collection-sumologic-setup` job is completed. You should see the status of the job being `Completed`.
-```sh
-kubectl get jobs -n sumologic
-```
-
-(Optional) You can delete the setup job which will automatically delete the associated pod as well.
-```sh
-kubectl delete job collection-sumologic-setup -n sumologic
-```
-
-Next, you will set up [Fluentd](#deploy-fluentd).
-
-### Manual Source Creation and Setup
+## Installation Steps
 
-This is a manual alternative approach to the automatic script if you don't have API access or need customized configuration, such as reusing an existing collector.
+These steps require that no Prometheus exists. If you already have Prometheus installed select from the following options:
```

Comment:
These alternate "existing Prometheus" docs are all Helm related, right? I don't think we should include these links in the non-helm install doc

Reply:
Good catch!